src/assets/data/edition/series/1/section/5/op12/source-description.json

```diff
@@ -1000,7 +1000,7 @@
                     {
                         "item": "",
                         "itemLinkTo": {},
-                        "itemDescription": "Autograph von <a (click)=\"ref.selectSvgSheet({complexId: 'op12', sheetId: 'M_212_TF1'})\">„Der Tag ist vergangen“ M 212: Textfassung 1</a>→2",
+                        "itemDescription": "Autograph von „Der Tag ist vergangen“ M 212: <a (click)=\"ref.selectSvgSheet({complexId: 'op12', sheetId: 'M_212_TF1'})\">Textfassung 1</a>→<a (click)=\"ref.selectSvgSheet({complexId: 'op12', sheetId: 'M_212_TF2'})\">2</a>",
```

Comment:
Ich frage mich, ob wir den Link auf dem ganzen Ausdruck belassen sollten. Momentan ist jeweils nur "Textfassung x" verlinkt, was die optische Erfassung/Unterscheidbarkeit tendenziell etwas verschlechtert.

Reply:
Jetzt kapiere ich: Du hast ja gar nichts vorgeschlagen, nur die von mir schon geänderte Zeile kommentiert... Also, bitte alles so lassen!

Reply:
Ja, verstehe. Fände es semantisch aber vermutlich immer noch sinnvoller, den vollen Ausdruck zu verlinken. Zumal es in AWG II/2a dann viele Fälle gibt, in denen "einzige Textfassung" dann einfach wegfällt und somit auf der M-Nummer verlinkt werden muss.